benchmarks/benchmark_serving.py

```diff
@@ -81,6 +81,31 @@
 from eval_accuracy import eval_accuracy
 
 
+def str2bool(v: str) -> bool:
```

Comment:
I remember @yeandy had made the type conversion in xlml pipeline. Do we need this in benchmark script?

Reply:
I don't think we made a type conversion. `benchmark_serving.py` probably interprets this as a string "true" rather than a `True` boolean.

https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/inference/configs/maxtext_inference_gce_config.py#L132

Reply:
I see, does the original implementation works in the xlml? If yes, do we still need this conversion?

Reply:
We will still need this conversion.
The problem is when you pass in --warmup-first false, which is interpreted as True

The only way of not warmup is not use this flag. That's why the conversion helps here when a boolean flag need to sweep across True and False.

Warmup is just one of the examples here.